narwhals/dataframe.py

```diff
@@ -2602,10 +2603,10 @@ def collect(self) -> DataFrame[Any]:
             >>> lf
             ┌───────────────────────────────────────────────┐
             | Narwhals LazyFrame                            |
-            | Use `narwhals.to_native` to see native output |
+            | Use `df.to_native` to see native output |
             └───────────────────────────────────────────────┘
```

Comment:
like this we're shortening the outupt, so the line at the right is no longer aligned - if you modify the message, you should also modify `__repr__`

maybe it should also say `DataFrame.to_native`? or `self.to_native`? `df` here doesn't refer to anything

Reply:
May I suggest

>Use `.to_native` method to see native output

?